crates/wasi-common/src/old/snapshot_0/sys/windows/hostcalls_impl/fs.rs

```diff
@@ -11,7 +11,7 @@ use crate::old::snapshot_0::sys::hostcalls_impl::fs_helpers::PathGetExt;
 use crate::old::snapshot_0::wasi::{self, WasiError, WasiResult};
 use log::{debug, trace};
 use std::convert::TryInto;
-use std::fs::{File, Metadata, OpenOptions};
+use std::fs::{self, File, Metadata, OpenOptions};
```

Comment:
Is this change only targeting the old snapshot?

Reply:
Whoops, `git rebase` went wrong. It should be fine now.